railties/lib/rails/tasks/initializers.rake

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 
+require "rails/command"
+require "active_support/deprecation"
+
 desc "Print out all defined initializers in the order they are invoked by Rails."
```

Comment:
Could you please remove this `desc`? It would remove this deprecated rake task from the thas list(`rake -T`)

Reply:
I know we've removed documentation before from other things, but I don't think that's a helpful habit or the best approach.

Reply:
@kaspth if it's deprecated, and the command is now documented under `rails`-proper, I don't see why we'd keep this?

Reply:
:+1: to remove

Reply:
Ah yeah, that's right 👍